### PR TITLE
Add blockdev_mirror_install for RHEL installation

### PR DESCRIPTION
**Pull request by zhencliu**
Signed-off-by: Zhenchao Liu <zhencliu@redhat.com>

ID: 1878441

### CODE REVIEW

**Comment by zhencliu** [comment]
hi @YongxueHong @aliang123 , this patch supports blockdev_mirror_install test, can you please help to review it, thanks.

File: shared/cfg/guest-os/Linux/RHEL.cfg [hunk 1]
```diff
@@ -5,7 +5,7 @@
         modprobe_module = acpiphp
     block_hotplug:
         modprobe_module = acpiphp
-    unattended_install, check_block_size..extra_cdrom_ks,svirt_install, with_installation, blockdev_commit_install, blockdev_snapshot_install:
+    unattended_install, check_block_size..extra_cdrom_ks,svirt_install, with_installation, blockdev_commit_install, blockdev_snapshot_install, blockdev_mirror_install:
```
Comment (YongxueHong) [inline shared/cfg/guest-os/Linux/RHEL.cfg:8]: Suggest to support it at the case level. thanks.

**Comment by zhencliu** [comment]
hi @aliang123 , please hold on review, refer to @YongxueHong comment, we can add installation settings on case level, I will make some changes, will ping you when code is ready, thank you.

**Comment by zhencliu** [comment]
Close this request referring to @YongxueHong 's comments, this is not required, I will add installation support in test case, thanks.

**Pull request closed without merging**